src/test/scala/com/microsoft/hyperspace/index/HybridScanTest.scala

```diff
@@ -565,17 +565,16 @@ class HybridScanTest extends QueryTest with HyperspaceSuite {
       var deletedFilesList: Seq[Seq[String]] = Nil
       val nodes = planWithHybridScan collect {
         case p @ Filter(
-              Not(In(attr, deletedFileNames)),
+              Not(expr),
               LogicalRelation(fsRelation: HadoopFsRelation, _, _, _)) =>
           // Check new filter condition on lineage column.
+          val attr = expr.children.head.asInstanceOf[Attribute]
+          val deletedFile = expr.children.last.asInstanceOf[Literal].toString
```

Comment:
Can we match exact instead of `expr.children`?
```
        case p @ Filter(
              Not(EqualTo(left: Attribute, right: Literal)),
              LogicalRelation(fsRelation: HadoopFsRelation, _, _, _)) =>
          // Check new filter condition on lineage column.
          val colName = left.name
          val deletedFile = right.toString
```